### PR TITLE
Add doc of time skew on physical nodes

### PR DESCRIPTION
**Pull request by Andrewmatilde**
Signed-off-by: Andrewmatilde <davis6813585853062@outlook.com>

### CODE REVIEW

File: docs/simulate-time-chaos-on-physical-nodes.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+| timeOffset | string   | Specifies the length of time offset.                                                                                                                                                                                                                                                                                                                                        | None                 | Yes      | `-5m`                                   |
+| clockIds   | []string | Specifies the ID of clock that will be offset. See the [<clock>clock_gettime</clock> documentation](https://man7.org/linux/man-pages/man2/clock_gettime.2.html) for details.                                                                                                                                                                                                | `["CLOCK_REALTIME"]` | No       | `["CLOCK_REALTIME", "CLOCK_MONOTONIC"]` |
```
Comment (WangXiangUSTC) [inline docs/simulate-time-chaos-on-physical-nodes.md:72]: what are `<clock>` and `</clock>` mean? looks like label

File: docs/simulate-time-chaos-on-physical-nodes.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+| pid        | string   | Specifies the mode of the experiment. The mode options include `one` (selecting a random Pod), `all` (selecting all eligible Pods), `fixed` (selecting a specified number of eligible Pods), `fixed-percent` (selecting a specified percentage of Pods from the eligible Pods), and `random-max-percent` (selecting the maximum percentage of Pods from the eligible Pods). | None                 | Yes      | `1`                                     |
```
Comment (WangXiangUSTC) [inline docs/simulate-time-chaos-on-physical-nodes.md:73]: the description looks wrong.

File: docs/simulate-time-chaos-on-physical-nodes.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        printf("Time : %lld.%.9ld\n", (long long)ts.tv_sec, ts.tv_nsec);
+        sleep(10);
+    }
+}
+EOF
+
+gcc -o get_time ./time.c
+```
+
+Then execute get_time and try to attack it.
```
[the remark this answers is not in the document]
Comment (Andrewmatilde) [reply docs/simulate-time-chaos-on-physical-nodes.md:65]: OK.

File: docs/simulate-time-chaos-on-physical-nodes.md (hunk 1 above)
Comment (Andrewmatilde) [reply docs/simulate-time-chaos-on-physical-nodes.md:72]: I will remove it.It doesn't work here.

File: docs/simulate-time-chaos-on-physical-nodes.md (hunk 2 above)
Comment (Andrewmatilde) [reply docs/simulate-time-chaos-on-physical-nodes.md]: Fixed.

File: docs/simulate-time-chaos-on-physical-nodes.md (hunk 1 above)
Comment (Andrewmatilde) [reply docs/simulate-time-chaos-on-physical-nodes.md]: Removed.

File: docs/simulate-time-chaos-on-physical-nodes.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Andrewmatilde) [reply docs/simulate-time-chaos-on-physical-nodes.md:65]: Fixed.

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

File: docs/simulate-time-chaos-on-physical-nodes.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+---
+title: Simulate time chaos
```
Comment (hfxsd) [inline docs/simulate-time-chaos-on-physical-nodes.md:2]:
```suggestion
title: Simulate time chaos
summary: This document describes how to use Chaosd to simulate a time offset scenario.
```

File: docs/simulate-time-chaos-on-physical-nodes.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        sleep(10);
+    }
+}
+EOF
+
+gcc -o get_time ./time.c
+```
+
+Then execute get_time and try to attack it.
+Such as:
```
Comment (hfxsd) [inline docs/simulate-time-chaos-on-physical-nodes.md:66]:
```suggestion
The following is an example:
```

File: docs/simulate-time-chaos-on-physical-nodes.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+```bash
+chaosd attack clock -p $PID -t 11s
+```
+
+### Configuration description of simulating disk read load
```
Comment (hfxsd) [inline docs/simulate-time-chaos-on-physical-nodes.md:72]:
```suggestion
### Configurations of simulating disk read load
```

Comment (Andrewmatilde) [reply docs/simulate-time-chaos-on-physical-nodes.md]: Fixed.

File: docs/simulate-time-chaos-on-physical-nodes.md (hunk 5 above)
Comment (Andrewmatilde) [reply docs/simulate-time-chaos-on-physical-nodes.md]: Fixed.

File: docs/simulate-time-chaos-on-physical-nodes.md (hunk 4 above)
Comment (Andrewmatilde) [reply docs/simulate-time-chaos-on-physical-nodes.md:2]: Fixed.

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]

**Pull request merged**